xarray/backends/zarr.py

```diff
@@ -1281,6 +1336,7 @@ def open_zarr(
     use_zarr_fill_value_as_mask=None,
     chunked_array_type: str | None = None,
     from_array_kwargs: dict[str, Any] | None = None,
+    cache_array_keys: bool = False,
```

Comment:
```suggestion
    cache_members: bool = False,
```

to match `ZarrStore`?